manifests/params.pp

```diff
@@ -333,6 +349,10 @@
       $agent_restart_command = undef
       $unavailable_runmodes = ['cron', 'systemd.timer']
     }
+    'Archlinux': {
+      $agent_restart_command = "/usr/bin/systemctl reload-or-restart ${service_name}"
+      $unavailable_runmodes = ['cron']
```

Comment:
Ok, that's alright then - the user can always override `unavailable_runmodes` if necessary.